packages/flutter_tools/lib/src/build_system/filecache.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package flutter_tools;
+
```

Comment:
Either here or in file_hash_store.dart (or maybe even both), include instructions on regenerating the .dart files.

Reply:
Added instructions on regenerating protos